package.json

```diff
@@ -2,13 +2,19 @@
   "name": "@dfinity/hardware-wallet-cli",
   "version": "0.3.0",
   "description": "A CLI to interact with the Internet Computer App on Ledger Nano S/X devices.",
-  "main": "./build/index.js",
+  "main": "./dist/index.js",
+  "files": [
+    "dist/index.js",
+    "README.md",
+    "LICENSE"
+  ],
   "scripts": {
     "format": "prettier --write .",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "tsc --build",
+    "build": "npm ci && node rmdir.mjs && node esbuild.mjs",
```

Comment:
I would not run `npm ci` each time you `build`

Reply:
It was during development. I was changing version and wanted to do it at once.

I will also remove the "old-build"

Reply:
done